doc/docs.md

```diff
@@ -36,17 +36,17 @@ You need `git`, and a C compiler like `tcc`, `gcc`, `clang` or `msvc`:
 ```bash
 git clone https://github.com/vlang/v
 cd v
-make.bat -tcc
+make -tcc
 ```
-NB: You can also pass one of `-gcc`, `-msvc`, `-clang` to `make.bat` instead,
+NB: You can also pass one of `-gcc`, `-msvc`, `-clang` to `make` instead,
 if you do prefer to use a different C compiler, but -tcc is small, fast, and
 easy to install (V will download a prebuilt binary automatically).
```

Comment:
NB: V will download and use a prebuilt TCC C-compiler by default because it's small, fast and easy to install. However, you can use a different C compiler by appending `-gcc`, `-msvc`, `-clang` to `make`.